themes/bootstrap3/js/common.js

```diff
@@ -290,6 +290,34 @@ function keyboardShortcuts() {
   }
 }
 
+/**
+ * Setup facets
+ * @returns {undefined}
```

Comment:
Looks like this was copy-pasted from another function definition in the file. For now I've removed both @returns annotations since they don't really tell us anything useful. We may want to work on standardizing JS commenting in the future, though.